solution/backend/regulations/logout.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+
+
+def eua_logout(request):
+    id_token = request.session.get('oidc_id_token')
+    # get the domain url from the request and add /login to the end
+    logout_redirect_url = request.build_absolute_uri('/') + settings.STAGE_ENV + '/logout'
+
+    if '//logout' in logout_redirect_url:
+        logout_redirect_url = logout_redirect_url.replace('//logout', '/logout')
```

Comment:
You can remove the conditional here. If `//logout` is _not_ in the variable, the `replace` function does nothing. Same result.